src/utils/persistAnswers.js

```diff
@@ -1,26 +1,31 @@
 export const persistAnswer = (
-  title,
-  selectedAnswer,
-  correctAnswer
+    title,
+    selectedAnswer,
+    correctAnswer
 ) => {
-  typeof window !== 'undefined' &&
-    localStorage.setItem(
-      title,
-      JSON.stringify({ selectedAnswer, correctAnswer })
-    );
+    typeof window !== 'undefined' &&
+        localStorage.setItem(
+            title,
+            JSON.stringify({ selectedAnswer, correctAnswer })
+        );
 };
 
 export const getPersistedAnswer = title => {
-  const item =
-    typeof window !== 'undefined' &&
-    localStorage.getItem(title);
-  if (!item) {
-    return { selectedAnswer: null, correctAnswer: null };
-  }
-  return JSON.parse(item);
+    const item =
+        typeof window !== 'undefined' &&
+        localStorage.getItem(title);
+    if (!item) {
+        return { selectedAnswer: null, correctAnswer: null };
+    }
+    return JSON.parse(item);
 };
 
 export const clearPersistedAnswer = title => {
-  typeof window !== 'undefined' &&
-    localStorage.removeItem(title);
+    typeof window !== 'undefined' &&
+        localStorage.removeItem(title);
 };
+
+export const clearAllPersistedAnswer = title => {
```

Comment:
This doesn't need to take a title argument, right?

Reply:
(making facepalm) no...no it doesnt...